portfolio/src/main/webapp/script.js

```diff
@@ -52,4 +52,23 @@ function addName(name) {
     console.log('Adding name to DOM...');
     const nameDiv = document.getElementById('name-container');
     nameDiv.innerText = name;
+}
+
+function fetchComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
+        console.log('Fetching comments...');
+        console.log(comments);
+        const commentListElement = document.getElementById('comment-container');
+        commentListElement.innerHTML = '';
+        for (index = 0; index < comments.length; index ++) {
+            commentListElement.appendChild(
```

Comment:
Out of curiosity, do we not need an empty `<ul>` element inside this div to contain all the newly created `<li>` elements?

Reply:
since testing it with the datastore PR, it doesnt seem like it. but i will find out.

Reply:
most documentation ive seen so far creates a <ul> element to contain the <li> elements. i havent found anything that says you cant just skip creating the <ul> element, but it seems standard to create an empty <ul> element, so i will add one.

Reply:
did this get added?